### PR TITLE
SFD-3257 code coverage for login

### PR DESCRIPTION
**Pull request by ArifNhs**
SFD-3257 code coverage for login and readme file update

### CODE REVIEW

File: application/app/pom.xml [hunk 1]
```diff
@@ -109,15 +109,38 @@
...
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-devtools</artifactId>
+        <optional>true</optional>
+    </dependency>
+    <dependency>
+        <groupId>org.mockito</groupId>
+        <artifactId>mockito-junit-jupiter</artifactId>
+        <version>3.11.2</version>
+        <scope>test</scope>
+    </dependency>
```
Comment (daonhs) [inline application/app/pom.xml:129]: For test coverage wouldnt we want to use the plugin JaCoCo

**Review by daonhs — COMMENTED** [review]
Wouldnt hurt to add the JaCoCo plugin to generate coverage reports and will ensure we have high code coverage for later developments

**Review by gwaardenburg — APPROVED** [review]

File: application/app/pom.xml (hunk 1 above)
Comment (jope1) [reply application/app/pom.xml:129]: Sounds like an interesting thing to plug in. Lets do it.  Can we also please add a section in the readme.txt to say how to run it?

**Review by jope1 — APPROVED** [review]
Good stuff.

File: application/app/pom.xml (hunk 1 above)
Comment (ArifNhs) [reply application/app/pom.xml:129]: I will add that in the upcoming stories

**Pull request merged**